mysql_mimic/session.py

```diff
@@ -277,6 +278,13 @@ async def _show_interceptor(self, expression: exp.Expression) -> AllowedResult:
             return await self._query_info_schema(select)
         return None
 
+    async def _describe_interceptor(self, expression: exp.Expression) -> AllowedResult:
```

Comment:
Might be cleaner to implement this in sqlglot: https://github.com/tobymao/sqlglot/blob/main/sqlglot/dialects/mysql.py

I can help with that

Reply:
That make sense. I will apply this change.

Reply:
Could you please grant me access to not wait approval for CI pipeline?

Reply:
I think we'll need @kelsin 's help to enable that